client/src/app/Log.js

```diff
@@ -252,11 +296,15 @@ export default class Log extends PureComponent {
 // helpers /////////////////////////////////
 
 function selectText(element) {
-  var range, selection;
+  let range, selection;
 
   selection = window.getSelection();
   range = document.createRange();
   range.selectNodeContents(element);
   selection.removeAllRanges();
   selection.addRange(range);
 }
+
+function hasSelection() {
+  return window.getSelection().toString() != '';
```

Comment:
Any specific reason for soft comparison?

Reply:
No (: